buildout.cfg

```diff
@@ -21,3 +22,7 @@ eggs =
 recipe = zc.recipe.testrunner
 eggs =
     Products.ZCatalog
+
+[omelette]
```

Comment:
I am not sure this is wanted

Reply:
I always like the omelette to make navigating between dependency package files easier. But let me know if I should remove this.